.github/workflows/build.yml

```diff
@@ -0,0 +1,34 @@
+name: Test
+
+on: [push]
+
+jobs:
+  build:
```

Comment:
Should we rather call this job `lint`? Since we're not really building anything here.